src/main/java/com/iexec/worker/tee/post/compute/PostComputeAppRunner.java

```diff
@@ -0,0 +1,88 @@
+/*
+ * Copyright 2022 IEXEC BLOCKCHAIN TECH
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.iexec.worker.tee.post.compute;
+
+import com.iexec.common.replicate.ReplicateStatusCause;
+import com.iexec.common.worker.api.ExitMessage;
+import com.iexec.worker.tee.post.compute.utils.EnvUtils;
+import feign.FeignException;
+import lombok.extern.slf4j.Slf4j;
+
+import static com.iexec.common.replicate.ReplicateStatusCause.POST_COMPUTE_TASK_ID_MISSING;
+import static com.iexec.common.replicate.ReplicateStatusCause.POST_COMPUTE_FAILED_UNKNOWN_ISSUE;
+import static com.iexec.common.worker.result.ResultUtils.RESULT_TASK_ID;
+import static com.iexec.worker.tee.post.compute.worker.WorkerApiManager.getWorkerApiClient;
+
+@Slf4j
+public class PostComputeAppRunner {
+
+    /**
+     * Exits:
+     * - 0: Success
+     * - 1: Failure; Reported cause (known or unknown)
+     * - 2: Failure; Unreported cause since reporting issue failed
+     * - 3: Failure; Unreported cause since missing taskID context
+     */
+    public void start() {
+        log.info("Tee worker post-compute started");
+
+        String chainTaskId = null;
+
+        try {
+            chainTaskId = EnvUtils.getEnvVarOrThrow(RESULT_TASK_ID, POST_COMPUTE_TASK_ID_MISSING);
+        } catch (PostComputeException e) {
+            log.error("TEE post-compute cannot go further without taskID context");
+            System.exit(3);
+        }
+
+        Integer exitCode = null;
+        ReplicateStatusCause exitCause = null;
+
+        try {
+            final PostComputeApp postComputeApp = createPostComputeApp(chainTaskId);
+            postComputeApp.runPostCompute();
+            log.info("TEE post-compute completed");
+            exitCode = 0;
```

Comment:
`System.exit` here ?
It should allow to remove the following check on `exitCode == null`

Reply:
Sure, see https://github.com/iExecBlockchainComputing/tee-worker-post-compute/pull/45/commits/028c4fb0226161f7ded4fd153bffcd3d11bd06bc.

Reply:
Thanks for thee update